docker-compose-example.yaml

```diff
@@ -0,0 +1,17 @@
+redash:
+  build: .
+  ports:
+    - "5000:5000"
+    - "9001:9001"
+  links:
+    - redis
+    - postgres
+  stdin_open: true
+redis:
+  image: redis:3.0
+  ports:
+    - "6379:6379"
+postgres:
+  image: postgres:9.4
```

Comment:
Currently the bootstrap script installs Postgres 9.3 and Redis 2.8.17. Let's keep using those for now to maintain parity between environments. If we end up needing a feature from a newer version in the future, we will be able to migrate easily using Docker. Hopefully by then many users will migrate to Docker based setup.